.github/workflows/publish.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        name: [ ChimeProd, ChimeSDKMeetingsProdIAD, ChimeSDKMeetingsProdPDX, ChimeSDKMeetingsProdFRA, ChimeSDKMeetingsProdSIN ]
+        name: [ ChimeProd, ChimeSDKMeetingsProdIAD, ChimeSDKMeetingsProdPDX, ChimeSDKMeetingsProdFRA, ChimeSDKMeetingsProdSIN, ChimeSDKMeetings_ChimeSDKMediaPipelinesProdIAD, Chime_ChimeSDKMediaPipelinesProdIAD ]
```

Comment:
Do we want to specify region for ChimeSDKMeetings? Seems like not regional and only `ChimeSDKMediaPipelinesProdIAD` is regional.

Suggest updating the name for ChimeSDKMeetings to include region.

Reply:
sorry, not sure I understand the comment. Could you clarify?

Reply:
Sorry for not being clear, basically was checking if below is more appropriate:
`ChimeSDKMeetings<Region>_ChimeSDKMediaPipelinesProdIAD`? Currently, the region is missing.